scripts/replace-package.js

```diff
@@ -17,19 +16,28 @@ if (!parsed || !parsed.groups.from || !parsed.groups.to) {
   throw new Error('Invalid format for REPLACE_PACKAGE');
 }
 
-const { from, to } = parsed.groups;
+function resolveTag(from, to) {
+  return execSync(`npm dist-tag ls ${from}@${to} | awk -F ': ' '/^${to}/ {print \$2}'`).toString().trim();
+}
+
+const { from, to: _to } = parsed.groups;
+
+// npm install doesn't seem to do anything if you're updating a
+// package-lock.json file that already has the dep to a tag like nightly, but it
+// does do something if you change it to the exact version.
+const to = _to === 'nightly' ? resolveTag(from, _to) : _to;
```

Comment:
At this point you might as well just make the script driver-specific? :)

Reply:
The thought did cross my mind 😆